xml/article_sap_trento.xml

```diff
@@ -0,0 +1,1052 @@
+    </section>
+
+<!--
+ toms 2021-11-19:
+ Do we have specific terms which needs to be explained in a terminology section?
+ -->
+
+    <section xml:id="sec-trento-software-requirements">
+        <title>Requirements</title>
+        <para>
+            This section describes the requirements for the &t.server; and its &t.agent;s, be it
+            software, hardware, or network.
+        </para>
+        <section>
+            <title>&t.server; requirements</title>
+            <para>
+                The suggested physical resources for running all the &t.server; components are 2GB of RAM
+                and two CPU cores.
+            </para>
+            <para>
+                The &t.server; is intended to run in many ways, depending on users' already existing
+                infrastructure, but it's designed to be cloud-native and OS agnostic.
+                As such, our default installation method provisions a minimal, single node, K3S &k8s;
+                cluster to run its various components in Linux containers.
+                The &t.server; can be installed on these services:
+            </para>
+            <remark>toms 2021-12-06: taken from "Draft - Various Tidbits regarding Productization"
+                (https://confluence.suse.com/x/tgCnMg)</remark>
+            <itemizedlist>
+                <listitem>
+                    <para>
+                        &k8s;
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>RKE1 (&rancher.k8s.engine; version 1)</para>
+                </listitem>
+                <listitem>
+                    <para>RKE2</para>
+                </listitem>
+                <listitem>
+                    <para>any other CNCF certified &k8s;</para>
+                </listitem>
+            </itemizedlist>
+        </section>
+        <section>
+            <title>&t.agent; requirements</title>
+            <para>
+                The resource footprint of the &t.agent; should not impact the performance of the
+                host it runs on.
+            </para>
+            <para>
+            The &t.agent; component, on the other hand, needs to interact with a number
+            of low-level system components which are part of the &productname; distribution.
+            These could in theory also be installed and configured on other distributions providing
+            the same functionalities, but this use case is not within the scope of the active development.
+            </para>
+        </section>
+        <section>
+            <title>Network requirements</title>
+            <para/>
+            <remark>toms 2021-12-02: which ports do we need?</remark>
+            <itemizedlist>
+                <listitem>
+                    <para>
+                        <remark>toms 2021-12-10: needs clarification:</remark>
+                        If you use &k8s;, ensure there is network access between the Trento component
```

Comment:
If? Everybody uses K8s, so, just say: Ensure that ...

Reply:
Corrected.